RHEL/6/input/profiles/ftp-server.xml

```diff
@@ -1,6 +1,6 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
-<description override="true">This profile is for FTP servers.</description>
+<title override="true">FTP Server Profile (vsftpd)</title>
+<description override="true">This is a profile for the vsftpd FTP server</description>
```

Comment:
Meh, I have actually removed it, based on random check of other profile (pci-dss one, I guess). We should probably make some simple pass over all descriptions, to make them consistently useful :). But I don't want to pollute this PR.